src/sessions/softphone-session-handler.ts

```diff
@@ -66,42 +438,130 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
       });
     }
 
+    /* make sure we store the session with the conversation state */
+    if (!this.conversations[session.conversationId]) {
+      this.conversations[session.conversationId] = { session } as any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=8384644)
```suggestion
      this.conversations[session.conversationId] = { session } as unknown;
```